projects/testing-library/src/lib/models.ts

```diff
@@ -186,6 +186,21 @@ export interface RenderComponentOptions<ComponentType, Q extends Queries = typeo
    * })
    */
   componentProviders?: any[];
+  /**
+   * @description
+   * A collection of imports to override a standalone component's imports with.
+   *
+   * @default
+   * undefined
+   *
+   * @example
+   * const component = await render(AppComponent, {
+   *   componentImports: [
+   *     MockChildComponent
+   *   ]
+   * })
+   */
+  componentImports?: (Type<any> | any[])[];
```

Comment:
I believe this could change in the future, and I would like to be able to just do that without justifications 😅
Maybe we should mark it as experimental as well in the TS Docs?

```suggestion
  ɵcomponentImports?: (Type<any> | any[])[];
```

Reply:
Sure np I can do that. Makes sense. I think the docs for `standalone` and `imports` mark them as experimental.